src/bulkActionPopup.js

```diff
@@ -122,40 +123,38 @@ export default function bulkActionPopupFactory(config) {
 
         // renders the component
         .on('render', function() {
-            var self = this;
-            var $element = this.getElement();
-            var $reason;
+            const $element = this.getElement();
 
             initModal({
                 disableEscape: true,
                 width: this.config.single && !this.config.deniedResources.length && !this.config.reason ? 600 : 800
             });
 
             if (_.isObject(this.config.categoriesSelector)) {
-                $reason = $element.find('.reason').children('.categories');
+                const $reason = $element.find('.reason').children('.categories');
                 this.config.categoriesSelector.render($reason);
             }
 
             $element
-                .on(namespaceHelper.namespaceAll('selected.cascading-combobox', _ns), function(e, reasons) {
+                .on(namespaceHelper.namespaceAll('selected.cascading-combobox', _ns), (e, reasons) => {
                     state.reasons = reasons;
-                    if (self.config.allowShortcuts) {
+                    if (this.config.allowShortcuts) {
                         // ensure the keyboard navigation is taking care of the possible new fields
                         initNavigator();
                     }
-                    self.trigger('change', state);
+                    this.trigger('change', state);
                 })
-                .on(namespaceHelper.namespaceAll('change', _ns), 'textarea', function() {
-                    state.comment = $(this).val();
-                    self.trigger('change', state);
+                .on(namespaceHelper.namespaceAll('change', _ns), 'textarea', (e) => {
```

Comment:
our code style says "no parenthesis around single parameter for arrow functions"

Reply:
@jsconan We don't have this rule in the Style guide https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/79037064/TAO+Frontend+Coding+guide
and in .eslintrc.json
Should I add it in .eslintrc.json ?
`"arrow-parens" :                ["error", "as-needed"]`
https://eslint.org/docs/rules/arrow-parens

Reply:
Ok, no worries. We use to apply the rule, but it is not expressed in the coding guide.
This is not a blocker, so I will merge the PR